src/pages/developers/tutorials.tsx

```diff
@@ -377,69 +316,37 @@ const TutorialPage = ({
+          <div className="mt-0 p-12 text-center">
             <Emoji text=":crying_face:" className="my-8 text-5xl" />
-            <OldHeading>
+            <h2 className="mb-8 mt-12 leading-xs">
               <Translation id="page-developers-tutorials:page-tutorial-tags-error" />
-            </OldHeading>
+            </h2>
             <Text>
               <Translation id="page-developers-tutorials:page-find-wallet-try-removing" />
             </Text>
-          </Box>
+          </div>
         )}
         {filteredTutorials.map((tutorial) => {
           return (
-            <Flex
-              as={BaseLink}
-              textDecoration="none"
-              flexDirection="column"
-              justifyContent="space-between"
-              fontWeight="normal"
-              color="text"
-              boxShadow="0px 1px 1px var(--eth-colors-tableItemBoxShadow)"
-              mb="px"
-              padding={8}
-              w="full"
-              _hover={{
-                textDecoration: "none",
-                borderRadius: "base",
-                boxShadow: "0 0 1px var(--eth-colors-primary-base)",
-                bg: "tableBackgroundHover",
-              }}
+            <LinkFlex
+              className="mb-px w-full flex-col justify-between border-b p-8 text-border no-underline duration-100 hover:rounded-sm hover:bg-background-low hover:shadow-table-box-hover"
               key={tutorial.href}
               href={tutorial.href ?? undefined}
-              hideArrow
             >
-              <Flex
-                justifyContent="space-between"
-                mb={{ base: 8, md: -4 }}
-                alignItems="flex-start"
-                flexDirection={{ base: "column", md: "initial" }}
-              >
-                <Text
-                  color="text"
-                  fontWeight="semibold"
-                  fontSize="2xl"
-                  me={{ base: 0, md: 24 }}
-                  _after={{
-                    ms: 0.5,
-                    me: "0.3rem",
-                    display: tutorial.isExternal ? "inline-block" : "none",
-                    content: `"↗"`,
-                    transform: flipForRtl,
-                    transitionProperty: "all",
-                    transitionDuration: "0.1s",
-                    transitionTimingFunction: "ease-in-out",
-                    fontStyle: "normal",
-                  }}
+              <Flex className="mb-8 flex-col items-start justify-between gap-y-4 md:-mb-4 md:flex-row">
+                <h4
+                  className={`relative me-0 text-body after:ml-2 after:inline-block after:italic after:transition-all after:duration-100 after:ease-in-out after:content-['↗'] md:me-24 ${tutorial.isExternal ? "after:inline-block" : "after:hidden"}`}
                 >
                   {tutorial.title}
-                </Text>
-                <Badge variant="secondary">
+                </h4>
+                <Badge
```

Comment:
Similar as the previous comment. We are replacing the Badge with the `ui/Tag`. Im adding a note about this in the GH guidance.

Since this work is done in #13976 we could:
- revert this part
- or use the `ui/Tag` component with `<Tag variant="outline">`